include/extractor/profile_properties.hpp

```diff
@@ -12,7 +12,7 @@ struct ProfileProperties
 {
     ProfileProperties()
         : traffic_signal_penalty(0), u_turn_penalty(0), continue_straight_at_waypoint(true),
-          use_turn_restrictions(false), left_hand_driving(false)
+          use_turn_restrictions(false), left_hand_driving(false), max_speed(0)
```

Comment:
Care to elaborate?

We don't expose the ProfilesProperties type - we bind it for Lua, but if we set a default value here users don't have to set one in the profiles. Should be compatible from what I can see?